src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -9218,6 +9267,78 @@ export class StudyViewPageStore
             return isMixedReferenceGenome(this.queriedPhysicalStudies.result);
         }
     }
+    private _clinicalEventFilters = observable.array<DataFilter>([], {
+        deep: false,
+    });
+
+    @computed get clinicalEventFilters(): DataFilter[] {
+        return this._clinicalEventFilters.slice();
+    }
+
+    @action
+    public setClinicalEventTypeFilter(filters: DataFilter[]) {
+        this._clinicalEventFilters.replace(filters);
+    }
+
+    @action
+    public resetClinicalEventTypeFilter() {
+        this._clinicalEventFilters.clear();
+    }
+
+    @computed get clinicalEventTypeFiltersRawStrings(): string[][] {
+        const filters = _.map(this.clinicalEventFilters, filterList =>
```

Comment:
coupla things here.  1. js arrays have map/reduce/filter methods built in so you don't need to use lodash for simple cases this.  ```this.clinicalEventFilters.map((filterList)=>...)``` will work